ee/vellum_ee/workflows/display/tests/workflow_serialization/test_basic_api_node_serialization.py

```diff
@@ -220,122 +220,6 @@ def test_serialize_workflow(vellum_client):
-                "inputs": [
-                    {
-                        "id": "5e892e5b-0004-4a04-bd2e-1ea9e0e5d3f9",
-                        "key": "node_input",
-                        "value": {
-                            "rules": [
-                                {
-                                    "type": "NODE_OUTPUT",
-                                    "data": {
-                                        "node_id": "6783c80f-5fc0-4712-a251-ce59d3c14ff2",
-                                        "output_id": "88082704-42bc-453f-a02a-40754453546b",
-                                    },
-                                }
-                            ],
-                            "combinator": "OR",
-                        },
-                    }
-                ],
-                "display_data": {"position": {"x": 400.0, "y": -50.0}},
-                "base": {
-                    "name": "FinalOutputNode",
-                    "module": ["vellum", "workflows", "nodes", "displayable", "final_output_node", "node"],
-                },
-                "definition": None,
-            },
-            {
-                "id": "3f3ffc50-b156-48ac-b5f3-f68cb05c2b90",
-                "type": "TERMINAL",
-                "data": {
-                    "label": "Final Output",
-                    "name": "status_code",
-                    "target_handle_id": "0c98c306-b519-40d7-8b05-321b1dfd7f11",
-                    "output_id": "44ea8d75-e2a8-4627-85b1-8504b65d25c9",
-                    "output_type": "NUMBER",
-                    "node_input_id": "14345321-7e6b-4e2a-918a-7a5b0064f047",
-                },
-                "inputs": [
-                    {
-                        "id": "14345321-7e6b-4e2a-918a-7a5b0064f047",
-                        "key": "node_input",
-                        "value": {
-                            "rules": [
-                                {
-                                    "type": "NODE_OUTPUT",
-                                    "data": {
-                                        "node_id": "6783c80f-5fc0-4712-a251-ce59d3c14ff2",
-                                        "output_id": "f687c0a1-b63a-4c5c-b6ef-472c6108ae4b",
-                                    },
-                                }
-                            ],
-                            "combinator": "OR",
-                        },
-                    }
-                ],
-                "display_data": {"position": {"x": 400.0, "y": -300.0}},
-                "base": {
-                    "name": "FinalOutputNode",
-                    "module": ["vellum", "workflows", "nodes", "displayable", "final_output_node", "node"],
-                },
-                "definition": None,
-            },
-        ],
-        final_output_nodes,
-        ignore_order=True,
-    )
-
     # AND each edge should be serialized correctly
     serialized_edges = workflow_raw_data["edges"]
     assert not DeepDiff(
```

Comment:
DevinAI remove assertions about edges from these basic tests - we won't have any in the future for these one node workflows